CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ Changelog for NeoFS Node
 - The correct role parameter to invocation (#3127)
 - nil pointer error for `storage sanity` command (#3151)
 - Process designation event of the mainnet RoleManagement contract (#3134)
+- Problem that SN pays for `putContainerSize`, now tx send as notary and proxy pay for it (#3167)
```

Comment:
"Storage nodes running out of GAS because `putContainerSize` was not paid for by proxy"